src/instructlab/model/list.py

```diff
@@ -29,6 +29,7 @@
 )
 @click.option(
     "--list-checkpoints",
+    help="List the trained model checkpoints and all existing models.",
```

Comment:
since it's a description of the effect of this particular option, perhaps you could say smth like "Also list checkpoints [in addition to existing models]."

Reply:
Changed it to this